### PR TITLE
Updated code to handle limited value in config.yaml

### PR DESCRIPTION
**Pull request by KunalJadhav5**
# Handle limited value in config.yaml

```
(venv) daxa-kunal@kunal-ubu-test-20240215-155301:~/kunal_pebblo/pebblo/pebblo/app/config$ cat config.yaml
logging:
  level: debug
(venv) daxa-kunal@kunal-ubu-test-20240215-155301:~/kunal_pebblo/pebblo/pebblo/app/config$ cd ../../../
(venv) daxa-kunal@kunal-ubu-test-20240215-155301:~/kunal_pebblo/pebblo$ pebblo --config pebblo/app/config/config.yaml
Downloading topic, entity classifier models ...
Initializing topic classifier ...
Initializing topic classifier ... done
Initializing entity classifier ...
Initializing entity classifier ... done
Pebblo server starting ...
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 10/10 [00:08<00:00,  1.22it/s]
INFO:     Started server process [2119]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://localhost:8000 (Press CTRL+C to quit)
```

### CODE REVIEW

File: pebblo/app/config/config.py [hunk 1]
```diff
@@ -55,16 +55,19 @@ def load_config(path) -> dict:
...
         try:
-            with open(con_file, "r") as output:
-                cred_json = yaml.safe_load(output)
-                parsed_config = Config.parse_obj(cred_json)
+            with open(path, "r") as output:
+                cred_yaml = yaml.safe_load(output)
+                # Replace missing fields with default values
+                for key in conf_obj.dict().keys():
+                    if key not in cred_yaml:
+                        cred_yaml[key] = conf_obj.dict()[key]
```
[the remark this answers is not in the document]
Comment (KunalJadhav5) [reply pebblo/app/config/config.py:64]: yes it would.

[the remark this answers is not in the document]
Comment (KunalJadhav5) [reply pebblo/app/config/config.py:64]: yes it would

[the remark this answers is not in the document]
Comment (KunalJadhav5) [reply pebblo/app/config/config.py:64]: working as expected. Resolving conversation.

File: tests/app/config/test_config_validation.py [hunk 2]
```diff
@@ -105,16 +106,34 @@ def test_reports_config_validate(setup_and_teardown):
...
 
-    # Test with non-existent output directory
-    config = {
-        "format": "pdf",
-        "renderer": "xhtml2pdf",
-        "outputDir": "/non/existent/directory",
-    }
-    validator = ReportsConfig(config)
+
+def test_classifier_config_validate():
```
Comment (shreyas-damle) [inline tests/app/config/test_config_validation.py:110]: Can you add unit test for incomplete config file?

**Review by shreyas-damle — APPROVED** [review]

**Review by Raj725 — APPROVED** [review]
LGTM

**Pull request merged**